src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
```

Comment:
you can remove this file as well and recreate it if you need it in the future.

Reply:
removed